test/move_effect_healing_wish.c

```diff
@@ -0,0 +1,77 @@
+#include "global.h"
+#include "test_battle.h"
+
+ASSUMPTIONS
+{
+    ASSUME(gBattleMoves[MOVE_HEALING_WISH].effect == EFFECT_HEALING_WISH);
+    ASSUME(gBattleMoves[MOVE_LUNAR_DANCE].effect == EFFECT_HEALING_WISH);
+}
+
+#define TEST_MAX_HP (100)
+
+SINGLE_BATTLE_TEST("Healing Wish causes the user to faint and fully heals the replacement")
+{
+    GIVEN {
+        ASSUME(B_HEALING_WISH_SWITCH >= GEN_5);
+        PLAYER(SPECIES_GARDEVOIR); // Faster than Wobbufeet.
```

Comment:
I'd say instead of a comment saying that a Pokémon should be faster, maybe make sure they're faster with some `Speed(...);` calls?

It doesn't really matter much, but it's possible somebody in their own project changes the base speeds of these Pokémon and break the test if we rely on implicit speeds.

Reply:
Done